app/api/v1/endpoints/prediction.py

```
@@ -0,0 +1,48 @@
+            published_name=settings.MODEL_NAME,
+            image_data=image_data
+        )
+
+        # 예측 결과 처리, 확률로 정렬하고 상위 3개 선택
+        top_predictions = sorted(results.predictions, key=lambda p: p.probability, reverse=True)[:3]
+        
+        # JSON 응답 생성
+        return [PredictionResult(tag_name=pred.tag_name, probability=pred.probability) for pred in top_predictions]
+    
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Prediction error: {str(e)}")  # 예외 시 상세 정보 포함
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)